src/components/LocalizedLink/LocalizedLink.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Link } from 'gatsby';
+
+import getLocalizedPath from '../../i18n/getLocalizedPath';
```

Comment:
My opinion is that if the `getLocalizedPath` method should be available outside of `i18m` (isn't only internal), it should be exported from `src/i18n/index.js` and you should import it from ``src/i18n` just like we do with modules.

Reply:
:+1: